scripts/cicd/verify-gather-logs-operations.py

```diff
@@ -37,6 +37,7 @@
     'free-stg',
     'starter-us-east-1',
     'starter-us-east-2',
+    'starter-us-west-1',
```

Comment:
We have now also added starter-ca-central-1 to the list of starter clusters.

Reply:
ok let me add that one too while we're at it...